server/apps/distributed-app/docs/modules/ROOT/pages/configure/cassandra.adoc

```diff
@@ -215,4 +215,7 @@ If disabled, Lightweight transactions will no longer be used upon messages opera
 As message flags updates relies so far on a read-before-write model, it exposes yourself to data races leading to
 potentially update loss. Better performance are expected
 by turning it off. Reads performed as part of write transaction are also performed with a relaxed consistency.
+
+| mailrepository.strong.consistency
+| Optional. Boolean, defaults to true. Allows to not using lightweight transactions in CassandraMailRepository.
```

Comment:
Allows not to use lightweight transactions

Reply:
Maybe say also that we instead implement an indempotent behaviour (duplicates are overriden, missing entries upon deletes are ignored)